ui-tests/src/test/java/org/openmrs/reference/AddPatientAppointmentTest.java

```diff
@@ -48,7 +47,6 @@ public void addPatientAppointmentTest() throws Exception {
         manageProviderSchedulesPage.enterMaximumTimeValue("09", "30");
         manageProviderSchedulesPage.clickOnEndTimeButton();
         manageProviderSchedulesPage.enterService(SERVICE_NAME);
-        manageProviderSchedulesPage.clickOnEndTimeButton();
```

Comment:
Why is this line removed.

Reply:
Its 

> Why is this line removed.

Its a bug!

After simulating the test locally, I noticed that manageProviderSchedulesPage.clickOnEndTimeButton(); should come before  manageProviderSchedulesPage.enterService(SERVICE_NAME); based on the page design. And the method is already called  [here](https://github.com/openmrs/openmrs-distro-referenceapplication/blob/f1e0047531a97d08601ebff401050e7eb4df85fa/ui-tests/src/test/java/org/openmrs/reference/AddPatientAppointmentTest.java#L49) and calling it [again ](https://github.com/openmrs/openmrs-distro-referenceapplication/blob/f1e0047531a97d08601ebff401050e7eb4df85fa/ui-tests/src/test/java/org/openmrs/reference/AddPatientAppointmentTest.java#L51) is making the test to break in master build as seen in the logs shared above!